LayoutTests/fast/box-decoration-break/box-decoration-break-parsing.html

```diff
@@ -30,6 +30,19 @@
 shouldBeEqualToString("e.style.getPropertyValue('-webkit-box-decoration-break')", "");
 shouldBe("computedStyle.getPropertyValue('-webkit-box-decoration-break')", "'slice'");
 
+e.style.boxDecorationBreak = "slice";
+shouldBe("e.style.getPropertyValue('box-decoration-break')", "'slice'");
+shouldBe("computedStyle.getPropertyValue('box-decoration-break')", "'slice'");
+
+e.style.boxDecorationBreak = "clone";
+shouldBe("e.style.getPropertyValue('box-decoration-break')", "'clone'");
+shouldBe("computedStyle.getPropertyValue('box-decoration-break')", "'clone'");
+
+e.style.boxDecorationBreak = "";
+e.style.boxDecorationBreak = "invalid";
+shouldBeEqualToString("e.style.getPropertyValue('box-decoration-break')", "");
+shouldBe("computedStyle.getPropertyValue('box-decoration-break')", "'slice'");
+
```

Comment:
WPT already covers the unprefixed parts, we don't really need this. Maybe we could add `test_invalid_value` to the WPT for the last case.

Reply:
Once this bot run is done. I'll fix things. Thanks.

Reply:
https://wpt.fyi/results/css/css-break?label=master&label=experimental&aligned&q=box-decoration-break

I could add to this one indeed.
https://github.com/web-platform-tests/wpt/blob/57c9cc80fb/css/css-break/parsing/box-decoration-break-invalid.html